test/test_batch_consistency.py

```diff
@@ -69,6 +69,13 @@ def test_contrast(self):
         waveform = torch.rand(2, 100) - 0.5
         _test_batch(F.contrast, waveform, enhancement_amount=80.)
 
+    def test_sliding_window_cmn(self):
+        waveform = torch.randn(2, 1024) - 0.5
+        _test_batch(F.sliding_window_cmn, waveform, center=True, norm_vars=True)
+        _test_batch(F.sliding_window_cmn, waveform, center=True, norm_vars=False)
+        _test_batch(F.sliding_window_cmn, waveform, center=False, norm_vars=True)
+        _test_batch(F.sliding_window_cmn, waveform, center=False, norm_vars=False)
```

Comment:
We should definitely use `pytest.mark.parametrize` to simplify this.

Reply:
You can try `subTest` in `unittest`.

Reply:
Our test runner does not understand `subTest`. The test name does not show up in the execution result, so `subTest` does not add any value despite more code and indentation.